Gemfile

```diff
@@ -32,7 +33,11 @@ gem 'rdf'
 gem 'rdf-rdfxml'
 gem 'rdf-vocab'
 gem 'recaptcha', require: 'recaptcha/rails'
+gem 'redis'
```

Comment:
Why explicitly bundle these three redis gems, instead of letting sidekiq bundle its dependencies?

Reply:
we don't need to explicitly specify 'redis' here, but the other gems are required for initialization.
'redis-rails' as it implements the cache store adapter for redis_store.
'redis-namespace' as it let's us use the 'sidekiq' namespace.